voltdbclient/url.go

```diff
@@ -25,6 +25,7 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
+		// need go version >= 1.8 to use this method.
```

Comment:
I've added manuel getPort() method for old versions with your test cases.  & removed Port usage of url package.